hr_employee_operating_unit/security/security.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+    <record id="ir_rule_hr_emp_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="hr.model_hr_employee"/>
+        <field name="domain_force">['|',('operating_unit_ids','=',False),
+                                        ('operating_unit_ids','in',[g.id for g in user.operating_unit_ids])]</field>
+        <field name="name">Employee operating units Allowed</field>
+        <field eval="1" name="perm_unlink"/>
+        <field eval="1" name="perm_write"/>
+        <field eval="1" name="perm_read"/>
+        <field eval="1" name="perm_create"/>
```

Comment:
I'm not sure about this but all the other modules only define perm_read to 1 on the record rule. Is there any reason to set all the perms to 1 ?